packages/services/api/src/modules/organization/providers/organization-manager.ts

```diff
@@ -459,6 +518,20 @@ export class OrganizationManager {
       reservedSlugs: reservedOrganizationSlugs,
     });
 
+    await this.auditLog.record({
+      eventType: 'ORGANIZATION_SETTINGS_UPDATED',
```

Comment:
TODO @TuvalSimha, Maybe lets not have a general "ORGANIZATION_SETTINGS_UPDATED" event, but instead specific events for each action e.g. `ORGANIZATION_SLUG_UPDATED`.

Reply:
@kamilkisiela @dotansimha  WDYT?